### PR TITLE
macaque.0.7.3 - via opam-publish

### PR DESCRIPTION
**Pull request by vasilisp**
Macaque (Macros for Caml Queries) is a DSL for OCaml, which produces
SQL requests from a comprehension syntax. Macaque can build queries
from simpler components, using phantom types to ensure safety.

---
* Homepage: https://github.com/ocsigen/macaque
* Source repo: https://github.com/ocsigen/macaque
* Bug tracker: https://github.com/ocsigen/macaque/issues

---

Pull-request generated by opam-publish v0.3.1

### CODE REVIEW

**Comment by dsheets** [comment]
Toggling because Travis missed this.

File: packages/macaque/macaque.0.7.3/files/fix-configure.patch [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+   esac
+ done
+ 
+-make configure CONFIGUREFLAGS="$*"
++if [ ! -e setup.exe ] || [ _oasis -nt setup.exe ] || [ setup.ml -nt setup.exe ] || [ configure -nt setup.exe ]; then
++  ocamlfind ocamlopt -o setup.exe setup.ml || ocamlfind ocamlc -o setup.exe setup.ml || exit 1
++  rm -f setup.cmi setup.cmo setup.cmx setup.o
++fi
++./setup.exe -configure "$@"
+ # OASIS_STOP
```
Comment (dsheets) [inline packages/macaque/macaque.0.7.3/files/fix-configure.patch]: Why is this file necessary? The library was just released. If a bugfix needs to be made, why not re-release the same version with the fix or release a new patch version 0.7.4?

Comment (vasilisp) [inline packages/macaque/macaque.0.7.3/files/fix-configure.patch]: Unfortunately, the patch had only appeared in `opam-repository` (1b707f34 that fixes installation on the BSDs). I personally only discovered the issue while preparing the PR.

The file is OASIS-generated, and OASIS from master does the right thing (ocaml/oasis@dd333ca67b32d6f6fad06331c20326f2389a5f61). I just updated Macaque accordingly (ocsigen/macaque@e18b7f71905c4f39e0fe97d90574adb75d371ad1). I can tag a 0.7.4 and re-submit if preferred.

Comment (dsheets) [inline packages/macaque/macaque.0.7.3/files/fix-configure.patch]: It would be very nice to release a patch-free version.

Comment (vasilisp) [inline packages/macaque/macaque.0.7.3/files/fix-configure.patch]: Sure. I am closing this, and another PR will follow. Thanks for the review!
